apps/docs/src/content/reference/core/use-task.mdx

```diff
@@ -0,0 +1,201 @@
+---
+category: '@threlte/core'
+name: useTask
+sourcePath: 'packages/core/src/lib/hooks/useTask.ts'
+order: 5.21
+---
+
+
+<Tip type="info">
+  Tasks are part of Threlte's *Task Scheduling System*.
+  For details on how to use stages and tasks, see the [scheduling
+  tasks](/docs/learn/basics/scheduling-tasks) page.
+</Tip>
+
+The hook `useTask` is used to create a
+[task](/docs/learn/basics/scheduling-tasks#tasks) that is used to run code on
+every frame.
+
+## Creating an Anonymous Task
+
+In its most basic form, `useTask` takes a function as its first argument. This
+function will be executed on every frame, starting on the next frame and
+receives the delta time representing the time since the last frame as its first
```

Comment:
```suggestion
function will be executed on every frame, starting on the next frame. It receives a `delta`, representing the time since the last frame as its
```